source/openapi-admin-v3.yaml

```diff
@@ -4205,6 +4407,98 @@ components:
+    CustomResolver:
+      type: object
+      required:
+        - _id
+        - function_id
+        - on_type
+        - field_name
+      properties:
+        _id:
+          type: string
+          description: The custom resolver's unique ID.
+        function_id:
+          type: string
+          description: The resolver function's unique ID.
+        on_type:
+          type: string
+          description: |-
+            The name of the resolver's parent type. This can be
+            `"Query"`, `"Mutation"`, or the name of a generated type if
+            this is a computed property.
+        field_name:
+          type: string
+          description: The name of the custom resolver field that appears in the GraphQL schema.
+        input_type:
+          oneOf:
+            - type: string
+            - type: object
+          description: |-
+            The type of the custom resolver's `input` parameter. This
+            can be a scalar, the name of an existing generated type, or
+            a custom JSON schema object. If undefined, the resolver does
+            not accept an input.
+        input_type_format:
+          type: string
+          enum:
+            - scalar
+            - scalar-list
+            - generated
+            - generated-list
+            - custom
+          description: |-
+            The kind of input type the custom resolver uses. This value
+            must agree with the value of `input_type`:
+
+            - A scalar input type must use `"scalar"` or `"scalar-list"`
+            - A generated input type must use `"generated"` or `"generated-list"`
+            - A custom input type must use `"custom"`
+
+            If undefined, the resolver does not accept an input.
+        payload_type:
+          oneOf:
+            - type: string
+            - type: object
+          description: |-
+            The type of the value returned by the custom resolver. This
+            can be a scalar, the name of an existing generated type, or
+            a custom JSON schema object. If undefined, the resolver
+            returns a `DefaultPayload` object.
+        payload_type_format:
+          type: string
+          enum:
+            - scalar
+            - scalar-list
+            - generated
+            - generated-list
+            - custom
+          description: |-
+            The kind of payload type the custom resolver uses. This value
+            must agree with the value of `payload_type`:
+
+            - A scalar payload type must use `"scalar"` or `"scalar-list"`
+            - A generated payload type must use `"generated"` or `"generated-list"`
+            - A custom payload type must use `"custom"`
+
+            If undefined, the resolver returns a `DefaultPayload` object.
+    ValidationLevel:
+      type: string
+      enum: ["STRICT", "OFF"]
+    ValidationAction:
+      type: string
+      enum: ["ERROR", "WARN"]
+    ValidationSettings:
```

Comment:
is this only for GraphQL? if so i'd consider renaming 'GraphQL Validation Settings'

Reply:
Sure - updated